README.md

```diff
@@ -291,26 +292,26 @@ Here's a big, complicated query. Note that Honey SQL makes no attempt to verify
              [:between :f.e 10 20]]
     :group-by [:f.a]
     :having [:< 0 :f.e]
-    :order-by [[:b.baz :desc] :c.quux [:f.a :nulls-first]
+    :order-by [[:b.baz :desc] :c.quux [:f.a :nulls-first]]
     :limit 50
     :offset 10}
 
 (sql/format *1 {:param1 "gabba" :param2 2})
-=> ["SELECT DISTINCT f.*, b.baz, c.quux, b.bla AS \"bla-bla\", NOW(), @x := 10
-     FROM foo AS f, baz AS b
```

Comment:
git log grepping suggests this was intended -- apparently Oracle doesn't like the AS keyword.

https://github.com/jkk/honeysql/commit/a4e9f0ff4697f